File: java/metodos/funcion_para_mostrar_agenda.java
package metodos;
public class funcion_para_mostrar_agenda {
    public static void mostrarAgenda(String[] contactos) {
        System.out.println("=== AGENDA DE CONTACTOS ===");
        for (int i = 0; i < contactos.length; i++) {
            System.out.println((i + 1) + ". " + contactos[i]);
        }
    }
    public static void main(String[] args) {
        String[] contactos = {"Juan", "Maria", "Carlos", "Ana"};
        mostrarAgenda(contactos);
    }
}

File: visual_basic/metodos/funcion_para_mostrar_agenda/funcion_para_mostrar_agenda.vb
Imports System
Module funcion_para_mostrar_agenda
    Sub Main(args As String())
        Dim contactos As New List(Of String) From {
            "Juan", "Maria", "Carlos", "Ana"
        }
        MostrarAgenda(contactos)
        Console.ReadKey()
    End Sub
    Sub MostrarAgenda(ByVal lista As List(Of String))
        Console.WriteLine("=== AGENDA ===")
        For Each contacto As String In lista
            Console.WriteLine(contacto)
        Next
    End Sub
End Module